browser/extensions/api/brave_shields_api.cc

```diff
@@ -82,6 +82,33 @@ ExtensionFunction::ResponseAction BraveShieldsAllowScriptsOnceFunction::Run() {
   return RespondNow(NoArguments());
 }
 
+BraveShieldsDisableSpeedreaderOnceFunction::~BraveShieldsDisableSpeedreaderOnceFunction() {
```

Comment:
I think this should move to a different file and API, still only exposed to the brave shields extension, so it would be exposed as `chrome.speedReader`.  Here's an example of one I did recently: https://github.com/brave/brave-core/pull/2678/commits/5a6657057040d092b6c7b5ff445a6bddb2893825
agree @bridiver?

Reply:
Also we should talk to #product first, because I think simple view just landed on Nightly or will very soon where we don't show any switches by default.  So this probably needs some UI rework.

Reply:
moved to a different file and api - rest of the logic remains within shields while design is in progress